x/stakeibc/keeper/unbonding_records.go

```diff
@@ -148,6 +148,11 @@ func (k Keeper) GetHostZoneUnbondingMsgs(ctx sdk.Context, hostZone types.HostZon
 	for _, validatorAddress := range utils.StringMapKeys(finalUnbondingsByValidator) { // DO NOT REMOVE: StringMapKeys fixes non-deterministic map iteration
 		undelegationAmount := sdk.NewCoin(hostZone.HostDenom, finalUnbondingsByValidator[validatorAddress])
 
+		// Ignore validators with a zero undelegation amount to prevent a failed transaction on the host
+		if undelegationAmount.IsZero() {
```

Comment:
Great addition @jstr1121!

Reply:
This is not what I have added FYI - I think someone else added this. :)